ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Err.scala

```diff
@@ -12,7 +12,7 @@ import com.daml.ledger.participant.state.v1.PackageId
   * Note that processSubmission can also fail with a protobuf exception,
   * e.g. https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/InvalidProtocolBufferException.
   */
-sealed trait Err extends RuntimeException with Product with Serializable
+abstract class Err extends RuntimeException with Product with Serializable
```

Comment:
This should be enough while preserving it's nature. Can you please test it?
```suggestion
sealed abstract class Err extends RuntimeException with Product with Serializable
```